packages/core/src/sass/add-or-manage.scss

```diff
@@ -272,6 +272,10 @@
     }
 }
 
+.dc-mobile-dialog {
+    height: 100%;
```

Comment:
@farrah-deriv What is the use of this here? Did you have any issue with `height: 100vh` which is already in `dc-mobile-dialog`?

Reply:
yes, vh takes the height of the address bar in mobile causing the tab content to be cropped when scrolling whereas 100% takes the viewport height minus the address bar